content/chainguard/chainguard-enforce/how-to-install-chainctl.md

```diff
@@ -84,17 +84,231 @@ You should receive output similar to the following.
   \____| |_| |_| /_/   \_\ |___| |_| \_|  \____|   |_|   |_____|
 chainctl: Chainguard Control
 
-GitVersion:    0.1.39
-GitCommit:     98f4b3bc8a1ef111777f797e8248b737643eedc6
+GitVersion:    0.1.42
+GitCommit:     badf10d2d89c83c6f1366008ffc3936d1e4c36f4
 GitTreeState:  clean
-BuildDate:     2022-12-05T14:04:10Z
-GoVersion:     go1.19.3
+BuildDate:     2022-12-12T22:10:55Z
+GoVersion:     go1.19.4
 Compiler:      gc
 Platform:      darwin/arm64
 ```
 
 If you received different output, check your bash profile to make sure that your system is using the expected PATH. 
 
+### Verifying the `chainctl` binary with Cosign and Rekor
```

Comment:
I think we can collapse this section a bit, I don't think we need the `chainctl version` command and write up anymore

```suggestion
### Verifying `chainctl` 
```

Reply:
Alternatively, we can pull this whole verifying section out into its own page, which would also allow for a **Prerequisites** section. What do you think?